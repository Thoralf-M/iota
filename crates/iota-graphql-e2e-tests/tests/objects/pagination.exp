processed 15 tasks

task 1, lines 7-20:
//# publish
created: object(1,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 5168000,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2, lines 22-23:
//# run Test::M1::create --args 0 @A
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 3, lines 25-26:
//# run Test::M1::create --args 1 @A
created: object(3,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 4, lines 28-29:
//# run Test::M1::create --args 2 @A
created: object(4,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 5, lines 31-32:
//# run Test::M1::create --args 3 @A
created: object(5,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 6, lines 34-35:
//# run Test::M1::create --args 4 @A
created: object(6,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 7, line 37:
//# create-checkpoint
Checkpoint created: 1

task 8, lines 39-49:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IDJiZjn7+fp4kQqozh0VY7lA6e064D7L7RZm8ls97KbfAQAAAAAAAAA="
          },
          {
            "cursor": "IE8EwRx+qeW6DSeGW3fVOnMEVQzKjDlTHY2Uy0Zk5SXqAQAAAAAAAAA="
          },
          {
            "cursor": "IGagAr53uUWmiMx6ukpPDip5KKMPo3TzmXJkJn7hN/ZGAQAAAAAAAAA="
          },
          {
            "cursor": "IKK8HGgBfHpanugpKVVK7cDNtZ83tVvUXlYIcx4kEAGsAQAAAAAAAAA="
          },
          {
            "cursor": "INyyEEQzyZg6+pRagtaV78gFeiD/s9mdV1ud1jC0rqHVAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 9, lines 51-61:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IDJiZjn7+fp4kQqozh0VY7lA6e064D7L7RZm8ls97KbfAQAAAAAAAAA="
          },
          {
            "cursor": "IE8EwRx+qeW6DSeGW3fVOnMEVQzKjDlTHY2Uy0Zk5SXqAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 10, lines 63-92:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IDJiZjn7+fp4kQqozh0VY7lA6e064D7L7RZm8ls97KbfAQAAAAAAAAA=",
            "node": {
              "address": "0x32626639fbf9fa78910aa8ce1d1563b940e9ed3ae03ecbed1666f25b3deca6df"
            }
          },
          {
            "cursor": "IE8EwRx+qeW6DSeGW3fVOnMEVQzKjDlTHY2Uy0Zk5SXqAQAAAAAAAAA=",
            "node": {
              "address": "0x4f04c11c7ea9e5ba0d27865b77d53a7304550cca8c39531d8d94cb4664e525ea"
            }
          },
          {
            "cursor": "IGagAr53uUWmiMx6ukpPDip5KKMPo3TzmXJkJn7hN/ZGAQAAAAAAAAA=",
            "node": {
              "address": "0x66a002be77b945a688cc7aba4a4f0e2a7928a30fa374f3997264267ee137f646"
            }
          },
          {
            "cursor": "IKK8HGgBfHpanugpKVVK7cDNtZ83tVvUXlYIcx4kEAGsAQAAAAAAAAA=",
            "node": {
              "address": "0xa2bc1c68017c7a5a9ee82929554aedc0cdb59f37b55bd45e5608731e241001ac"
            }
          },
          {
            "cursor": "INyyEEQzyZg6+pRagtaV78gFeiD/s9mdV1ud1jC0rqHVAQAAAAAAAAA=",
            "node": {
              "address": "0xdcb2104433c9983afa945a82d695efc8057a20ffb3d99d575b9dd630b4aea1d5"
            }
          }
        ]
      }
    },
    "obj_3_0": {
      "address": "0x4f04c11c7ea9e5ba0d27865b77d53a7304550cca8c39531d8d94cb4664e525ea"
    },
    "obj_5_0": {
      "address": "0x66a002be77b945a688cc7aba4a4f0e2a7928a30fa374f3997264267ee137f646"
    },
    "obj_6_0": {
      "address": "0xdcb2104433c9983afa945a82d695efc8057a20ffb3d99d575b9dd630b4aea1d5"
    },
    "obj_4_0": {
      "address": "0xa2bc1c68017c7a5a9ee82929554aedc0cdb59f37b55bd45e5608731e241001ac"
    },
    "obj_2_0": {
      "address": "0x32626639fbf9fa78910aa8ce1d1563b940e9ed3ae03ecbed1666f25b3deca6df"
    }
  }
}

task 11, lines 94-106:
//# run-graphql --cursors @{obj_5_0}
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IKK8HGgBfHpanugpKVVK7cDNtZ83tVvUXlYIcx4kEAGsAQAAAAAAAAA="
          },
          {
            "cursor": "INyyEEQzyZg6+pRagtaV78gFeiD/s9mdV1ud1jC0rqHVAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 12, lines 108-118:
//# run-graphql --cursors @{obj_4_0}
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "INyyEEQzyZg6+pRagtaV78gFeiD/s9mdV1ud1jC0rqHVAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 13, lines 120-130:
//# run-graphql --cursors @{obj_3_0}
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IDJiZjn7+fp4kQqozh0VY7lA6e064D7L7RZm8ls97KbfAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 14, lines 132-144:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IKK8HGgBfHpanugpKVVK7cDNtZ83tVvUXlYIcx4kEAGsAQAAAAAAAAA=",
            "node": {
              "address": "0xa2bc1c68017c7a5a9ee82929554aedc0cdb59f37b55bd45e5608731e241001ac"
            }
          },
          {
            "cursor": "INyyEEQzyZg6+pRagtaV78gFeiD/s9mdV1ud1jC0rqHVAQAAAAAAAAA=",
            "node": {
              "address": "0xdcb2104433c9983afa945a82d695efc8057a20ffb3d99d575b9dd630b4aea1d5"
            }
          }
        ]
      }
    }
  }
}
