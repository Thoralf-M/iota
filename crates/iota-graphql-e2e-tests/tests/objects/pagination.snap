---
source: external-crates/move/crates/move-transactional-test-runner/src/framework.rs
---
processed 15 tasks

task 1, lines 7-20:
//# publish
created: object(1,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 5168000,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2, lines 22-23:
//# run Test::M1::create --args 0 @A
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 3, lines 25-26:
//# run Test::M1::create --args 1 @A
created: object(3,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 4, lines 28-29:
//# run Test::M1::create --args 2 @A
created: object(4,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 5, lines 31-32:
//# run Test::M1::create --args 3 @A
created: object(5,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 6, lines 34-35:
//# run Test::M1::create --args 4 @A
created: object(6,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 7, line 37:
//# create-checkpoint
Checkpoint created: 1

task 8, lines 39-49:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IB/UEvhb5e0rP0ewRxAbf5Qp8FtMN5onjFBAoToYJh+EAQAAAAAAAAA="
          },
          {
            "cursor": "ICZ82g9/mWryqQRCxYfPYpDMFQpC7fy/PnKGdXFysszHAQAAAAAAAAA="
          },
          {
            "cursor": "IHu07/dp2hGCZtIICH0SWs38fQnYzcuh1Amm1/dSLdwXAQAAAAAAAAA="
          },
          {
            "cursor": "IIeCz6n8mbLG6HnuWhfqpOLUDhBTF3c809DyIrqLg10KAQAAAAAAAAA="
          },
          {
            "cursor": "IL/cf/9GNFJN/BGC36aBlmNtdJ3tEPOPNlduX4LIAvi0AQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 9, lines 51-61:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IB/UEvhb5e0rP0ewRxAbf5Qp8FtMN5onjFBAoToYJh+EAQAAAAAAAAA="
          },
          {
            "cursor": "ICZ82g9/mWryqQRCxYfPYpDMFQpC7fy/PnKGdXFysszHAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 10, lines 63-92:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IB/UEvhb5e0rP0ewRxAbf5Qp8FtMN5onjFBAoToYJh+EAQAAAAAAAAA=",
            "node": {
              "address": "0x1fd412f85be5ed2b3f47b047101b7f9429f05b4c379a278c5040a13a18261f84"
            }
          },
          {
            "cursor": "ICZ82g9/mWryqQRCxYfPYpDMFQpC7fy/PnKGdXFysszHAQAAAAAAAAA=",
            "node": {
              "address": "0x267cda0f7f996af2a90442c587cf6290cc150a42edfcbf3e7286757172b2ccc7"
            }
          },
          {
            "cursor": "IHu07/dp2hGCZtIICH0SWs38fQnYzcuh1Amm1/dSLdwXAQAAAAAAAAA=",
            "node": {
              "address": "0x7bb4eff769da118266d208087d125acdfc7d09d8cdcba1d409a6d7f7522ddc17"
            }
          },
          {
            "cursor": "IIeCz6n8mbLG6HnuWhfqpOLUDhBTF3c809DyIrqLg10KAQAAAAAAAAA=",
            "node": {
              "address": "0x8782cfa9fc99b2c6e879ee5a17eaa4e2d40e105317773cd3d0f222ba8b835d0a"
            }
          },
          {
            "cursor": "IL/cf/9GNFJN/BGC36aBlmNtdJ3tEPOPNlduX4LIAvi0AQAAAAAAAAA=",
            "node": {
              "address": "0xbfdc7fff4634524dfc1182dfa68196636d749ded10f38f36576e5f82c802f8b4"
            }
          }
        ]
      }
    },
    "obj_3_0": {
      "address": "0xbfdc7fff4634524dfc1182dfa68196636d749ded10f38f36576e5f82c802f8b4"
    },
    "obj_5_0": {
      "address": "0x267cda0f7f996af2a90442c587cf6290cc150a42edfcbf3e7286757172b2ccc7"
    },
    "obj_6_0": {
      "address": "0x1fd412f85be5ed2b3f47b047101b7f9429f05b4c379a278c5040a13a18261f84"
    },
    "obj_4_0": {
      "address": "0x7bb4eff769da118266d208087d125acdfc7d09d8cdcba1d409a6d7f7522ddc17"
    },
    "obj_2_0": {
      "address": "0x8782cfa9fc99b2c6e879ee5a17eaa4e2d40e105317773cd3d0f222ba8b835d0a"
    }
  }
}

task 11, lines 94-106:
//# run-graphql --cursors @{obj_5_0}
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IHu07/dp2hGCZtIICH0SWs38fQnYzcuh1Amm1/dSLdwXAQAAAAAAAAA="
          },
          {
            "cursor": "IIeCz6n8mbLG6HnuWhfqpOLUDhBTF3c809DyIrqLg10KAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 12, lines 108-118:
//# run-graphql --cursors @{obj_4_0}
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IIeCz6n8mbLG6HnuWhfqpOLUDhBTF3c809DyIrqLg10KAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 13, lines 120-130:
//# run-graphql --cursors @{obj_3_0}
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IHu07/dp2hGCZtIICH0SWs38fQnYzcuh1Amm1/dSLdwXAQAAAAAAAAA="
          },
          {
            "cursor": "IIeCz6n8mbLG6HnuWhfqpOLUDhBTF3c809DyIrqLg10KAQAAAAAAAAA="
          }
        ]
      }
    }
  }
}

task 14, lines 132-144:
//# run-graphql
Response: {
  "data": {
    "address": {
      "objects": {
        "edges": [
          {
            "cursor": "IIeCz6n8mbLG6HnuWhfqpOLUDhBTF3c809DyIrqLg10KAQAAAAAAAAA=",
            "node": {
              "address": "0x8782cfa9fc99b2c6e879ee5a17eaa4e2d40e105317773cd3d0f222ba8b835d0a"
            }
          },
          {
            "cursor": "IL/cf/9GNFJN/BGC36aBlmNtdJ3tEPOPNlduX4LIAvi0AQAAAAAAAAA=",
            "node": {
              "address": "0xbfdc7fff4634524dfc1182dfa68196636d749ded10f38f36576e5f82c802f8b4"
            }
          }
        ]
      }
    }
  }
}
